packages/docusaurus-theme-search-algolia/src/index.js

```diff
@@ -29,7 +29,10 @@ function theme(context) {
     baseUrl,
     siteConfig: {title, url, favicon},
   } = context;
-  const pagePath = path.resolve(__dirname, './theme/SearchPage/index.js');
+  const pageComponent = './theme/SearchPage/index.js';
```

Comment:
just wondering, wouldn't using `@theme/SearchPage` work?

Reply:
No, because we need resolved component path, so it will be used with `fs` further.